src/consensus/aft/raft.h

```diff
@@ -44,6 +44,9 @@ namespace aft
       // the highest matching index with the node that was confirmed
```

Comment:
The functional changes are in this file. The rest of the changes are only for testing purposes or for updating the API.